services.yaml

```diff
@@ -1753,13 +1753,47 @@ paths:
     post:
       tags:
         - Events
-      summary: Force Event Snapshot
+      summary: Force event snapshot
       description: Force snapshot on the next time the event occurs
       responses:
         200:
           description: OK
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/events/{event_id}/actions:
+    get:
+      tags:
+        - Events
+      summary: "Fetch event actions"
+      description: "Fetch event actions"
+      parameters:
+          - in: path
+            name: service_id
+            description: Environment to use
+            type: string
+            required: true
+          - in: path
+            name: event_id
+            description: Event to use
+            type: string
+            required: true
+          - in: query
+            name: type
+            description: Filter by specific type
+            type: string
+            required: false
```

Comment:
no need to put required:false. if it's not required simply remove the attribute altogether

Reply:
fixed